integrations/flink/prometheus_metadata.yaml

```diff
@@ -0,0 +1,212 @@
+platforms:
```

Comment:
The flink-task-manager-prometheus dashboard (https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/2de3d85fcd11d86dc3d7429075287ac189fbb88c/dashboards/flink/flink-task-manager-prometheus.json) uses the following metrics that are missing in here:

- flink_taskmanager_Status_JVM_GarbageCollector_G1_Old_Generation_Time
- flink_taskmanager_Status_JVM_GarbageCollector_G1_Young_Generation_Time
- flink_taskmanager_Status_JVM_GarbageCollector_G1_Old_Generation_Count
- flink_taskmanager_Status_JVM_GarbageCollector_G1_Young_Generation_Count

Should these metrics be included here?